### PR TITLE
Add field-level "Learn more" and "Example" links.

### PR DESCRIPTION
**Pull request by jrobbins**
Add links to Github wiki pages to give additional field-level help and examples.

pursuant to #878

### CODE REVIEW

**Comment by jrobbins** [comment]
Screenshot:
https://screenshot.googleplex.com/X4jJatwtZPS

Following one of the links will open a new tab on a destination such as:
https://github.com/GoogleChrome/chromium-dashboard/wiki/EditingHelp#interoperability-and-compatibility-risks-example

File: guideforms.py [hunk 1]
```diff
@@ -37,15 +37,28 @@
...
         label='', required=True,
         widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 500}),
-        help_text=('Provide a one sentence description followed by one or '
-                   'two lines explaining how this feature helps web '
-                   'developers.')),
+        help_text=
+        ('Provide a one sentence description followed by one or '
+         'two lines explaining how this feature helps web '
+         'developers. '
+         '<a target="_blank" href="'
```
Comment (jpmedley) [inline guideforms.py:58]:
```suggestion
         '<a target="_blank" href="'
         'https://github.com/GoogleChrome/chromium-dashboard/wiki/'
         'EditingHelp#summary-learn-more">Learn more</a>. '
         '<a target="_blank" href="'
```

File: guideforms.py [hunk 2]
```diff
@@ -37,15 +37,28 @@
...
         required=True, label='Feature',
         widget=forms.TextInput(attrs={'autocomplete': 'off'}),
-        help_text=('Capitalize only the first letter and the beginnings of '
-                   'proper nouns.')),
+        help_text=
+        ('Capitalize only the first letter and the beginnings of '
+         'proper nouns. '
+         '<a target="_blank" href="'
+         'https://github.com/GoogleChrome/chromium-dashboard/wiki/'
+         'EditingHelp#feature-name">Learn more</a>. '
```
Comment (jpmedley) [inline guideforms.py:45]:
```suggestion
         'EditingHelp#feature-name-learn-more">Learn more</a>. '
```

File: guideforms.py [hunk 3]
```diff
@@ -63,10 +76,15 @@
     'motivation': forms.CharField(
         label='Motivation', required=True,
         widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 1480}),
-        help_text=('Explain why the web needs this change.  It may be useful '
-                   'to describe what web developers are forced to do without '
-                   'it. When possible, include links to back up your claims '
-                   'in the explainer.')),
+        help_text=
+        ('Explain why the web needs this change.  It may be useful '
```
Comment (jpmedley) [inline guideforms.py:80]:
```suggestion
        ('Explain why the web needs this change. It may be useful '
```

File: guideforms.py [hunk 4]
```diff
@@ -63,10 +76,15 @@
...
+         'to describe what web developers are forced to do without '
+         'it. When possible, include links to back up your claims '
+         'in the explainer. '
```
Comment (jpmedley) [inline guideforms.py:83]:
```suggestion
         'it. When possible, add links to your explainer '
         'backing up your claims. '
```

File: guideforms.py (hunk 2 above)
Comment (jrobbins) [reply guideforms.py:45]: Why add that to the URL anchor?

The gihub wiki generates HTML for headings that have the words of the heading in the anchor.  So, the current heading "Feature name" makes an anchor named "feature-name".  I don't know how we would change it to have the "-learn-more" suffix other than by adding visible text to the heading, which does not sound desirable.

File: guideforms.py (hunk 1 above)
Comment (jrobbins) [reply guideforms.py:58]: Same comment.

**Pull request merged**